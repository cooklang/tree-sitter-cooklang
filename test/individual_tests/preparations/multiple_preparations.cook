Mix @onion{1}(peeled and finely chopped) and @garlic{2%cloves}(peeled and minced) into paste.