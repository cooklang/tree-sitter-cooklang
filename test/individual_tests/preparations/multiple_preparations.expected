(recipe 
  (step 
    (ingredient 
      (name) 
      (amount (quantity))
      note: )
    (ingredient 
      (name) 
      (amount (quantity) (units))
      note: (note_text)))